src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -76,6 +76,22 @@ function getDrupalValue(arr) {
   return null;
 }
 
+/**
+ * A very specific helper function that expects to receive an
+ * array with one item which is an object with a single `processed` property
+ *
+ */
+function getDrupalProcessed(arr) {
+  if (arr.length === 0) return null;
+  if (arr.length === 1)
+    return typeof arr[0].value === 'string'
+      ? unescapeUnicode(arr[0].processed)
+      : arr[0].value;
+  // eslint-disable-next-line no-console
+  console.warn(`Unexpected argument: ${arr.toString()}`);
+  return null;
+}
+
```

Comment:
This function is exactly the same as `getDrupalValue`. The only difference is one uses the `processed` field.

Do we really need both functions? When should we use one vs. the other? 

If we do need both, how about using an argument to switch between `processed` and `value`?

Reply:
When should we use one vs. the other?
Based on the cms team, we should be using `processed` if exists.

Maybe just keep the `getDrupalValue` function and check if `processed` exists first and return that one, otherwise return `value`